### PR TITLE
 solution to collatz_conjecture math problem

### PR DESCRIPTION
**Pull request by imahnama**

### CODE REVIEW

**Comment by imahnama** [comment]
[Collatz conjecture](https://trello.com/c/ABpwHS6N/9-collatz-conjecture)

File: Algorithms/collatz-conj.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+num = int(input("Enter a number: ")) #prompt user to enter a number
+def collatz_con(n):
```
Comment (bqevin) [inline Algorithms/collatz-conj.py]: let's talk about naming functions 💅

**Review by bqevin — CHANGES_REQUESTED** [review]

File: Algorithms/collatz-conj.py (hunk 1 above)
Comment (imahnama) [reply Algorithms/collatz-conj.py]: Listening

Comment (bqevin) [reply Algorithms/collatz-conj.py]: At least let the name define what the function does and it's always good practice to avoid the shortening of names.

**Review by bqevin — APPROVED** [review]

**Pull request merged**